### PR TITLE
fix #52290 kreiszeitung.de

### LINKED ISSUE #52290: kreiszeitung.de
### Issue URL (Ads)
 [https://www.kreiszeitung.de/lokales/diepholz/bassum-ort51127/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.kreiszeitung.de%2Flokales%2Fdiepholz%2Fbassum-ort51127%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.4.11
Browser: | Chrome
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>EasyList, <br/>Peter Lowe's Blocklist<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection, <br/>EasyPrivacy, <br/>Fanboy's Enhanced Tracking List, <br/>Fanboy Anti-Facebook List, <br/>Fanboy's Anti-thirdparty Fonts<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media, <br/>Fanboy's Social Blocking List<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Fanboy's Annoyances, <br/>Adblock Warning Removal List, <br/>I don't care about cookies<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion<br/><br/><b>Language-specific:</b><br/> AdGuard German

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#52290

### CODE REVIEW

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 1]
```diff
@@ -33,6 +33,7 @@ reddit.com#$#body.scroll-disabled { overflow: visible!important; position: stati
 reddit.com##body > div[id="container"] > div[class="App"] > div[class="AppMainPage"] > div[class="NavFrame"] > div[class="XPromoPopup active"]
 !
 scribd.com##.auto__doc_page_webpack_view_in_store_promo
+kreiszeitung.de#?#.id-SiteMain > div.id-g > div[class] > div.id-Container:has(> div.id-Container-content > div.id-LB > div[class^="id-LB"] > div[class^="id-Teaser"] > a[href^="https://itunes.apple.com/de/app/mediengruppe-kreiszeitung/"])
```
Comment (BlazDT) [inline AnnoyancesFilter/sections/mobile-app.txt]: Not shown in code. iOS does not support ext-CSS, so making it the search parameter is a bit contraproductive.

File: GermanFilter/sections/specific.txt [hunk 2]
```diff
@@ -66,6 +66,7 @@ welt.de##.c-page-container--has-commercials
 haz.de,ostsee-zeitung.de,sn-online.de,dnn.de,kn-online.de,ln-online.de,neuepresse.de,lvz.de##.pdb-div-gpt-footerad
 !
 radiosaw.de###oms_gpt_rectanglesidebar
+||kreiszeitung.de/images/tz/img/eurojackpot-*.png
```
Comment (BlazDT) [inline GermanFilter/sections/specific.txt]: Not shown in code.

File: AnnoyancesFilter/sections/mobile-app.txt (hunk 1 above)
Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/mobile-app.txt]: for example <a href="https://www.kreiszeitung.de/lokales/diepholz/bassum-ort51127/corona-physiotherapeuten-haben-weniger-patienten-arbeiten-aber-weiter-13612348.html">here<a>
<details>
<summary>Screenshot</summary>

</details>

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/mobile-app.txt]: > Not shown in code. iOS does not support ext-CSS, so making it the search parameter is a bit contraproductive.

It looks like that this element can not be removed safely at iOS in any case and the element looks the same on all platforms.

Comment (BlazDT) [reply AnnoyancesFilter/sections/mobile-app.txt:36]: OK.

File: GermanFilter/sections/specific.txt (hunk 2 above)
Comment (BlazDT) [reply GermanFilter/sections/specific.txt:69]: Please remove the rule meanwhile.

Comment (Sergey-Lyapin) [reply GermanFilter/sections/specific.txt]: done

**Review by BlazDT — APPROVED** [review]

**Pull request merged**